### PR TITLE
Implement NPC_WIDECRITICALWOUND

### PR DESCRIPTION
**Pull request by eppc0330**
* **Addressed Issue(s)**: #1525

* **Server Mode**: Renewal

* **Description of Pull Request**: 
 Implement mob skill `NPC_WIDECRITICALWOUND` to fix item option of `XM_Celine_Kimi_Card`(ID:31023)

### CODE REVIEW

File: src/map/status.cpp [hunk 1]
```diff
@@ -11445,8 +11445,6 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 				val1 = rnd()%ELE_ALL;
 			break;
 		case SC_CRITICALWOUND:
```
Comment (aleos89) [inline src/map/status.cpp:11447]: Was this case supposed to be removed too? Right now it's going to fall into SC_MAGICMIRROR right below.

File: db/re/skill_db.yml [hunk 2]
```diff
@@ -17953,6 +17953,41 @@ Body:
...
+      - Level: 6
+        Area: 14
+      - Level: 7
+        Area: 14
+      - Level: 8
+        Area: 14
+      - Level: 9
+        Area: 14
+      - Level: 10
+        Area: 2
```
Comment (Lemongrass3110) [inline db/re/skill_db.yml:17988]: Looks like a copy paste issue?

File: src/map/status.cpp (hunk 1 above)
Comment (Lemongrass3110) [reply src/map/status.cpp:11447]: Agree with aleos here. This looks incorrect.
If it should not decrease heal effectiveness, this case needs to be removed completely and also the references in the calculations in pc.cpp and skill.cpp need to be removed.

**Review by Lemongrass3110 — APPROVED** [review]

**Comment by Lemongrass3110** [comment]
@eppc0330 you updated the branch meanwhile. Did you have time to check the comments? Otherwise we cannot merge this PR.

**Comment by eppc0330** [comment]
> meanwhile

Sorry what should I do?

**Comment by eppc0330** [comment]
@Lemongrass3110 Hmm.. Im query. I have commented explaining your both two questions and you are asking again. So it means my answers are not explaining enough right? Or is there something other problem? Like my comments are not visible for other?

**Comment by Lemongrass3110** [comment]
> @Lemongrass3110 Hmm.. Im query. I have commented explaining your both two questions and you are asking again. So it means my answers are not explaining enough right? Or is there something other problem? Like my comments are not visible for other?

If you did not click "submit review" those comments/answers are marked "pending" and are not visible for others.

**Comment by eppc0330** [comment]
@Lemongrass3110 Ohhh I didn't know it for real. Thank you and sorry. So thats' why people not replying my comments...
Now I submit review. Please check my reviews.

**Comment by eppc0330** [comment]
@Lemongrass3110 
https://www.divine-pride.net/database/skill/749

About skill range, actually I don't know the exact reason either but just followed DP

File: src/map/status.cpp (hunk 1 above)
Comment (Lemongrass3110) [reply src/map/status.cpp:11447]: Can you please change the code like you copied it above?
```
		case SC_CRITICALWOUND:
			// Level 1 ~ 5 & 6 ~ 10 has different duration
			// Level 6 ~ 10 use effect of level 1 ~ 5
			val1 = 1 + ((val1-1)%5);
			val2 = 20*val1;
			break;
```
This way it is more readable and additionally it fits our plans for the future better.
Thank you!

**Review by Lemongrass3110 — APPROVED** [review]

File: src/map/status.cpp (hunk 1 above)
Comment (eppc0330) [reply src/map/status.cpp:11447]: Thanks for review. I ll do it soon when Im home

**Review by aleos89 — APPROVED** [review]

**Pull request merged**